requirements.txt

```diff
@@ -1,4 +1,4 @@
-torch>=1.9.0,<=1.12
+torch>=1.9.0,<=1.13.1
```

Comment:
can we set it to 1.13.*?

Reply:
I tried with 1.13 and got a security issue, I guess because of the nightly hack.
But apparently I can set other 1.13.X if I write X explicitely: 1.13.5 didnt raise the security issue despite not being released yet.

Should we stick to the latest stable released version (1.13.1) or set the requirements to a higher version in advance to not have to modify the requirements.txt next time ?
I tend to prefer the first (1.13.1) but I don't mind that much